pkg/node/chain_stub.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package node
+
+import (
+	"context"
+	"errors"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/postage/postagecontract"
+	"github.com/ethersphere/bee/pkg/settlement/swap/chequebook"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/go-sw3-abi/sw3abi"
+)
+
+// senderMatcherStub is a stub for p2p.SenderMatcher interface.
```

Comment:
maybe not stub but `noop` or something along those lines? this goes for wherever we use stub in the context of this PR